### ISSUE TITLE
No information of GitHub Actions documentation build errors

### ISSUE THREAD

**Issue by albert-github** [CGAL/cgal#7227]
During a `\build:v...` we see in e.g. #6702

![image](https://user-images.githubusercontent.com/5223533/215471886-6d158d4b-444e-43fc-b06d-cb7a9175122c.png)

Looks like that some information is not displayed.
In the we see in `.github/workflows/build_doc.yml` (at the end):
```
      - name: Post error
        env:
          ERRORMSG: ${{steps.build_and_run.outputs.DoxygenError}}
        uses: actions/github-script@v6
        if: ${{ failure() && steps.get_round.outputs.result != 'stop' }}
        with:
          script: |
            const error = process.env.ERRORMSG
            const msg = "There was an error while building the doc: \n"+error
            github.rest.issues.createComment({
              owner: "CGAL",
              repo: "cgal",
              issue_number: ${{ github.event.issue.number }},
              body: msg
            });
```
so looks like:
```
          ERRORMSG: ${{steps.build_and_run.outputs.DoxygenError}}
```
or
```
const error = process.env.ERRORMSG
```
is not working

**Comment by albert-github** [comment]
With https://github.com/CGAL/cgal/pull/7198#issuecomment-1413897048 I do see a descriptive error message.

Has this been fixed or is it a side effect of 48b07bef84deaa9d5e0b0ecebf7c94794371f600 or 798bd0898992726c991fc74090f688e8e66d2c3e?

**Comment by lrineau** [comment]
@SaillantNicolas Please check if we can improve the error reporting. I think the comment should always add the URL to the job log, so that the user can retrieve the complete error more easily.
